### ISSUE TITLE
Add licensing links to licensing module

### ISSUE THREAD

**Issue by LourensVeen** [esciencecenter-digital-skills/research-software-support#150]
#149 adds some links to Turing Way sections on licensing as further reading. Should be added to the licensing module too.

**Comment by c-martinez** [comment]
I just came across this project, which could be a nice addition to the licensing module:

https://ufal.github.io/public-license-selector/

**Comment by LourensVeen** [comment]
Hm, it claims that Apache 2 and LGPL v2.1 or later are incompatible, which isn't true because "LGPL v2.1 or later" is a disjunction of LGPLv2 and LGPLv3, and LGPLv3 is compatible with the Apache 2 license. Also, it would only be relevant if you copy-pasted them together or were distributing them as a whole, not if you just put your own Python script online.

I notice that [some guy has been complaining about GPL for libraries](https://github.com/ufal/public-license-selector/issues/11) as well...

But maybe it's just that it's written in CoffeeScript, and I drink tea.

**Comment by c-martinez** [comment]
Yes, maybe the content of it has room for improvement, but is it still a useful resource for the licensing module?

**Comment by LourensVeen** [comment]
Well, I don't see what it adds over choosealicense.com, except for bad advice...

**Comment by LourensVeen** [comment]
Actually, it has the compatibility stuff, so that's good in principle. But since it's not implemented correctly, it's just going to point people in the wrong direction, and that's worse than not having something.

Now I want to go make a license selector website...

**Comment by c-martinez** [comment]
Isn't it easier to fix/improve what is already there? The advantage that I see is that it gives an alternative to choosealicense.com -- different people might find one or the other easier to understand, so having multiple options sounds good to me (as long as they do not contradict each other). But if you don't feel it is useful, then feel free to ignore.

**Comment by LourensVeen** [comment]
I tried, see the link above, but it doesn't seem like they're very interested in what anybody else thinks...

**Comment by c-martinez** [comment]
Other licensing links I came across recently:
https://www.software.ac.uk/guide/choosing-open-source-license
https://everse.software/RSQKit/licensing_software#what-is-copyright-and-licensing

Worth adding them?
